crates/analytics/src/init.rs

```diff
@@ -0,0 +1,63 @@
+use ethui_broadcast::{InternalMsg, UIMsg};
+use tauri::{AppHandle, Manager};
+
+use crate::track_event;
+
+pub async fn init(handle: &AppHandle) {
+    
+    let analytics = crate::Analytics::instance();
+    handle.manage(analytics);
+    
+    
+    let handle = handle.clone();
+    tauri::async_runtime::spawn(async move {
+        receiver(handle).await;
+    });
+}
+
+async fn receiver(handle: AppHandle) -> ! {
+    let mut internal_rx = ethui_broadcast::subscribe_internal().await;
+    let mut ui_rx = ethui_broadcast::subscribe_ui().await;
+
+    loop {
+        tokio::select! {
+            internal_msg = internal_rx.recv() => {
+                if let Ok(msg) = internal_msg {
+                    use InternalMsg::*;
+                    
+                    match msg {
+                        PeerAdded => {
+                            let _ = track_event(&handle, "extension_connected", None);
```

Comment:
```suggestion
                            let _ = track_event(&handle, "peer_added", None);
```

this will actually emit events anytime we connect a new tab, so it's better to use a 1:1 mapping, since "extension_connected" could be interpreted in other ways